package.json

```diff
@@ -18,9 +18,8 @@
   "homepage": "https://github.com/mozilla/readability",
   "devDependencies": {
     "chai": "^2.1.*",
-    "chai-as-promised": "^4.3.*",
     "html": "0.0.*",
-    "mocha": "^2.2.*",
-    "readable-proxy": "1.3.*"
+    "jsdom": "^3.1.2",
+    "mocha": "^2.2.*"
```

Comment:
Nit: keep trailing comma unless package.json's spec is stupid and doesn't allow you to do that or whatever